openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -356,7 +356,11 @@ def schedules_for_bin(self):
             try:
                 week_highlights = get_week_highlights(user, enrollment.course_id, week_num)
             except CourseUpdateDoesNotExist:
-                continue
+                LOG.exception(
```

Comment:
This is also logged for a single recipient.  Can we keep log it once for a (course, week) tuple?

Reply:
Doesn't it also depend on the user? E.g. two users in different cohorts: one has access to week 7's content, the other one does not and just has 6 weeks of content. The former user will not hit this exception but the latter would.